homeassistant/components/overkiz/sensor.py

```diff
@@ -481,7 +481,12 @@ def native_value(self) -> StateType:
         """Return the value of the sensor."""
         state = self.device.states.get(self.entity_description.key)
 
-        if not state or not state.value:
+        if (
+            state is None
+            or state.value is None
+            or self.state_class != SensorStateClass.MEASUREMENT
+            and not state.value
+        ):
             return None
```

Comment:
We return `None` when:
- the state is `None`, or
- The sensor is not a MEASUREMENT sensor and the value is falsy.

Reply:
I think we should add a comment that the last case here is probably incorrect and should be fixed in a follow up.

Reply:
#105141